packages/graphback-core/src/plugin/ModelDefinition.ts

```diff
@@ -1,12 +1,20 @@
-import { GraphQLObjectType, GraphQLField } from "graphql";
+import { GraphQLObjectType } from "graphql";
 import { FieldRelationshipMetadata } from '../relationships/RelationshipMetadataBuilder';
 import { GraphbackCRUDGeneratorConfig } from "./GraphbackCRUDGeneratorConfig"
 
+/**
+ * Maps type fields to corresponding field in the database
+ */
+export type ModelField = {
```

Comment:
```suggestion
export type ModelFieldMap = {
```

Reply:
Good suggestion. 

I am still thinking about this and the field information we'd want to store in this map.

My recent thoughts pushed me to think that we'll need more than field column name in the database, additional information like column type might be needed so that an appropriate query in the database is performed. I am thinking of `ObjectID` for MongoDB filtering and `JSON` types for Postgres. 

A naive map could look lik: 

```ts
/**
Field descriptor in the database
**/
export type FieldDescriptor = {
    name: string, // could come from GraphQLField.name or later on from a @column(name: 'db_name') annotation or the @db annotation
    type: string // could come from GraphQLField.type or later on from @column(type: 'db_type') or the @db annotation
}

/**
 * Maps type fields to corresponding field in the database
 */
export type ModelField = {
  [key : string]: FieldDescriptor
}
```
It needs more thinking.

Reply:
I went with this thought. Let me know what you think.